src/xdocs/config_misc.xml

```diff
@@ -1243,9 +1243,25 @@ void foo(String aFooString,
           Rationale: Any source files and text files in general should
           end with a line separator to let other easily add new content
           at the end of file and "diff" command does not show previous lines as changed.
-          <br/>
+        </p>
+        <p>
           Example (line 36 should not be in diff):
-          <img src="https://cloud.githubusercontent.com/assets/812984/13894408/afd965b8-ed24-11e5-8bfd-e9edf56a6fe6.png" alt="example of diff"/>
+        </p>
+        <source>
+@@ -32,4 +32,5 @@ ForbidWildcardAsReturnTypeCheck.returnTypeClassNamesIgnoreRegex
+PublicReferenceToPrivateTypeCheck.name = Public Reference To Private Type
+
+StaticMethodCandidateCheck.name = Static Method Candidate
+-StaticMethodCandidateCheck.desc = Checks whether private methods should be declared as static.
+\ No newline at end of file
++StaticMethodCandidateCheck.desc = Checks whether private methods should be declared as static.
++StaticMethodCandidateCheck.skippedMethods = Method names to skip during the check.
+        </source>
+        <p>
+          It can also trick the VCS to report the wrong owner for such lines.
```

Comment:
I added this part from myself. I doubt that this can affect java code (usually there is a simple `}` at the end of the file) but, for example, property files can be affected.

Reply:
ok